extensions/MYTAditya/CommentPlus.js

```diff
@@ -0,0 +1,383 @@
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+            },
+          },
+          {
+            opcode: "commentReporter",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[INPUT] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "",
+              },
+            },
+          },
+          {
+            opcode: "commentBoolean",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "[INPUT] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.BOOLEAN,
+              },
+            },
+          },
+          {
+            opcode: "commentReporterPPP",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "// [COMMENT] [INPUT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanPPP",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "// [COMMENT] [INPUT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.BOOLEAN,
+              },
+            },
+          },
+          {
+            opcode: "commentReporterPPPP",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "// [COMMENT] [INPUT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.BOOLEAN,
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanPPPP",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "// [COMMENT] [INPUT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "",
+              },
+            },
+          },
+          {
+            opcode: "commentBoolenPlus",
```

Comment:
You misspelled "Boolean" so this doesn't do anything (see later comment for when it does do something)